packages/graphql/src/configs/rinkeby.js

```diff
@@ -14,6 +14,7 @@ export default {
   discovery: 'https://discovery.staging.originprotocol.com',
   growth: 'https://growth.staging.originprotocol.com',
   bridge: 'https://bridge.staging.originprotocol.com',
+  idenbtityServer: 'https://bridge.staging.originprotocol.com',
```

Comment:
why separate from bridge?  Seems redundant?

Reply:
It is mostly for convenience when testing on local host. It allows to point bridge to https://bridge.dev.originprotocol.com and use it as an attestation server that is already configured, while pointing identityServer to a local bridge server that stores identity in a local db.